src/mail/MailUtils.js

```diff
@@ -553,10 +553,25 @@ export function moveToInbox(mails: Mail[]): Promise<*> {
 	}
 }
 
-export function exportMails(mails: Mail[]): Promise<void> {
+/**
+ * convert a set of mail to a set of dataFiles in eml format
+ * @param mails array of mails to convert
+ * @returns {Promise<(DataFile|any)[]>}
+ */
+export function mailsToEmlDataFiles(mails: Mail[]): Promise<DataFile[]> {
```

Comment:
```suggestion
export function mailsToEmlDataFiles(mails: Array<Mail>): Promise<Array<DataFile>> {
```
Array type literals are soon to be deprecated.